LondonBritishLibrary/orient/BLorient13262.xml

```diff
@@ -0,0 +1,174 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BLorient13262" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Malkǝʾa Dāwit, Magical Prayers</title>
```

Comment:
Stricktly speaking, the hymn to David is not "Malkǝʾa Dāwit". Please check it up, harmonize with the record for work.